composer.json

```diff
@@ -5,13 +5,13 @@
     "keywords": ["ace", "code", "editor", "symfony", "form", "bundle", "javascript"],
     "license": "MIT",
     "require": {
-        "php": ">=7.3",
+        "php": ">=8.1",
         "twig/twig": "^1.34|^2.0|^3.0",
-        "symfony/framework-bundle": "^3.0|^4.0|^5.0",
-        "symfony/form": "^3.0|^4.0|^5.0",
-        "symfony/twig-bridge": "^3.0|^4.0|^5.0",
-        "symfony/asset": "^3.0|^4.0|^5.0",
-        "symfony/expression-language": "^3.0|^4.0|^5.0"
+        "symfony/framework-bundle": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/form": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/twig-bridge": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/asset": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/expression-language": "^3.0|^4.0|^5.0|^6.0"
```

Comment:
I think we can safely reduce the Symfony versions to `^5.4||^6`

Reply:
fixed